docs/testing/troubleshooting.md

```diff
@@ -75,6 +75,9 @@ The following steps should help troubleshoot any problems related to the **Autom
 
 [!INCLUDE [Teams support note](../includes/teams-support-note.md)]
 
+> [!NOTE]
+> There is a known issue that can prevent scripts stored in SharePoint from being displayed in the ribbon. This occurs when your admin turns off Exchange Web Services (EWS). Your SharePoint-based scripts are still accessible through the file dialog and are able to be run and edited.
```

Comment:
This is mostly true - EWS prevent scripts stored in SharePoint from being cached in a user's most recent scripts. During the session the user would see the SP script in their most recently used scripts, but once they close the workbook/end the session the SP script would disappear from the list.

Reply:
Updated. Does this version sound better?